src/block-management/block-holder.js

```diff
@@ -279,6 +268,7 @@ export default compose( [
 
 		return {
 			mergeBlocks( forward ) {
+				const { clientId } = ownProps;
```

Comment:
I think it might be useful to bring this destructuring assignment up (out of the returned literal), as it would dry up the code that follows (and reduce the diff). WDYT? I might be missing some context on why we are using `ownProps` as a receiver to access the `clientId` property each time.

Reply:
Ah, sorry it wasn't clear, the context is mentioned in the PR description, point No8. Basically, just wanted to match the code style of the `withDispatch` section in the web's counterpart of our block-holder: https://github.com/WordPress/gutenberg/blob/aafab3e71b9587f57bd23d5b818f32cd04929556/packages/block-editor/src/components/block-list/block.js#L689. Let me know if that doesn't make sense or it's perhaps buggy and we can revise.

Reply:
It wasn't a concern about bugginess, but rather more of a style nit. In light of your explanation, I think it makes sense. :+1: